torch/onnx/_internal/diagnostics/rules.yaml

```diff
@@ -82,3 +82,181 @@
+    text: FX pass converting torch.neg to torch.sigmoid.
+  full_description:
+    text: "A 'fx.Interpreter' based pass to convert all 'torch.neg' calls to 'torch.sigmoid' for
+      a given 'torch.fx.GraphModule' object."
+    markdown: |
+      A 'fx.Interpreter' based pass to convert all 'torch.neg' calls to 'torch.sigmoid' for
+      a given 'torch.fx.GraphModule' object.
+  message_strings:
+    default:
+      text: "Running 'convert-neg-to-sigmoid' pass on 'torch.fx.GraphModule'."
+  help_uri:
+  properties:
+    deprecated: false
+    tags: []
+
+
+- id: FXE0005
+  name: fx-ir-add-node
+  short_description:
+    text: ToDo, experimenting diagnostics, placeholder text.
+  full_description:
+    text: "ToDo, experimenting diagnostics, placeholder text."
+    markdown: |
+      ToDo, experimenting diagnostics, placeholder text.
+  message_strings:
+    default:
+      text: "ToDo, experimenting diagnostics, placeholder text."
+  help_uri:
+  properties:
+    deprecated: false
+    tags: []
+
+
+- id: FXE0006
+  name: atenlib-symbolic-function
+  short_description:
+    text: Op level tracking. ToDo, experimenting diagnostics, placeholder text.
+  full_description:
+    text: "ToDo, experimenting diagnostics, placeholder text."
+    markdown: |
+      ToDo, experimenting diagnostics, placeholder text.
+  message_strings:
+    default:
+      text: "ToDo, experimenting diagnostics, placeholder text."
+  help_uri:
+  properties:
+    deprecated: false
+    tags: []
+
+
+- id: FXE0007
+  name: atenlib-fx-to-onnx
+  short_description:
+    text: Graph level tracking. Each op is a step. ToDo, experimenting diagnostics, placeholder text.
+  full_description:
+    text: "ToDo, experimenting diagnostics, placeholder text."
+    markdown: |
+      ToDo, experimenting diagnostics, placeholder text.
+  message_strings:
+    default:
+      text: "ToDo, experimenting diagnostics, placeholder text."
+  help_uri:
+  properties:
+    deprecated: false
+    tags: []
+
+- id: FXE0008
+  name: fx-node-to-onnx
+  short_description:
+    text: Node level tracking. ToDo, experimenting diagnostics, placeholder text.
+  full_description:
+    text: "ToDo, experimenting diagnostics, placeholder text."
+    markdown: |
+      ToDo, experimenting diagnostics, placeholder text.
+  message_strings:
+    default:
+      text: "ToDo, experimenting diagnostics, placeholder text."
+  help_uri:
+  properties:
+    deprecated: false
+    tags: []
+
+- id: FXE0009
+  name: fx-frontend-dynamo-make-fx
+  short_description:
+    text: The make_fx + decomposition pass on fx graph produced from Dynamo, before ONNX export.
+  full_description:
+    text: "ToDo, experimenting diagnostics, placeholder text."
+    markdown: |
+      ToDo, experimenting diagnostics, placeholder text.
+  message_strings:
+    default:
+      text: "ToDo, experimenting diagnostics, placeholder text."
+  help_uri:
+  properties:
+    deprecated: false
+    tags: []
+
+
+- id: DIAGSYS0001
```

Comment:
Is there anything important that we should know before adding rules? If there is, will you write it into OVERVIEW.md? Or we could just follow the format in this yaml?

Reply:
I expect to do this later, but hopefully soon. https://github.com/microsoft/onnx-converters-private/issues/148